storage_policies/README.md

```diff
@@ -0,0 +1,266 @@
+# Storage policies
+
+This document describes where our production datasets are stored, how [object
+lifecycles] are configured, and how access permissions are managed.
+
+[object lifecycles]: https://cloud.google.com/storage/docs/lifecycle
```

Comment:
Not sure if we want to stick to inline links or allow both ref and inline types?

Reply:
Good catch, thanks! Fixed.